integration/tests/economy/test_economics.py

```diff
@@ -1126,3 +1125,51 @@ def test_iterative_failed_canceled_trx_with_out_of_gas(
         assert_profit(
             sol_balance_before - sol_balance_after, sol_price, token_diff, neon_price, web3_client.native_token_name
         )
+
+    def test_operator_profit_for_iterative_failed_canceled_trx(
+        self,
+        counter_contract: Contract,
+        web3_client,
+        neon_price,
+        accounts,
+        sol_price: float,
+        operator: Operator,
+    ):
+        sol_balance_before = operator.get_solana_balance()
+        token_balance_before = operator.get_token_balance(web3_client)
+        tx = web3_client.make_raw_tx(from_=accounts[0].address, tx_type=TransactionType.EIP_1559)
+
+        instruction_tx = counter_contract.functions.moreInstruction(0, 3000).build_transaction(tx)
+        receipt = web3_client.send_transaction(accounts[0], instruction_tx)
+        assert receipt["status"] == 1
+        gas_used = receipt["gasUsed"]
+
+        max_attempts = 10
+        attempt = 0
+        while attempt < max_attempts:
```

Comment:
I think we can have a loop here like that
while receipt["status"] != 0

and delete old test as current test will include everything from test_iterative_failed_canceled_trx_with_out_of_gas